jellyfin_kodi/jellyfin/connection_manager.py

```diff
@@ -283,21 +283,22 @@ def _convert_endpoint_address_to_manual_address(self, info):
         return None
 
     def _normalize_address(self, address):
-        # TODO: Try HTTPS first, then HTTP if that fails.
-        if '://' not in address:
-            address = 'http://' + address
-
         # Attempt to correct bad input
         url = urllib3.util.parse_url(address.strip())
 
-        if url.scheme is None:
-            url = url._replace(scheme='http')
-
-        if url.scheme == 'http' and url.port == 80:
-            url = url._replace(port=None)
+        # Default to using https
+        url = url._replace(scheme='https')
 
-        if url.scheme == 'https' and url.port == 443:
-            url = url._replace(port=None)
+        # Test if server is reachable over https
+        if self.API.get_public_info(url):
+            if url.scheme == 'https' and url.port == 443:
```

Comment:
Is the port stuff needed anymore?  Since it's just integrated with the URL field, it may not even need a check here

Reply:
I mostly did it this way to avoid urls such as http://foo:80/asdf, there's no practical *need* for it, just a log readability concern